opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -31,7 +34,8 @@
    *                   .setTag("lucky_number", 42);
    * }</pre>
    */
-  Span startTrace(String operationName);
+  Span startSpan(String operationName, Optional<Span> parent);
 
-  SpanPropagator getSpanPropagator();
+  <T> void inject(Span span, T carrier);
+  <T> Span join(String operationName, T carrier);
```

Comment:
only because there's no reference in the spec as to what it is. 

in https://github.com/opentracing/opentracing.github.io/issues/70 `injectSpan` had been shortened to `inject`, so i'd presumed the same for `join`.